mmv1/products/memorystore/Instance.yaml

```
@@ -706,3 +706,40 @@ properties:
           description:
             "Output only. Ports of the exposed endpoint."
           output: true
+  - name: 'gcsSource'
+    type: NestedObject
+    description: "GCS source for the instance."
+    immutable: true
+    ignore_read: true
+    conflicts:
+      - managedBackupSource
+    properties:
+      - name: 'uris'
+        type: Array
```

```suggestion
        type: Array
        is_set: true
```

We won't get diffs because these are ignore_read, but this will mean reordering doesn't trigger recreations.